components/building-highlights/building-highlights.component.jsx

```diff
@@ -10,7 +10,11 @@ const BuildingHighlights = () => {
   /**
    * Fill color for the Polygons
    */
-  const [fillColor, setFillColor] = useState("rgba(128, 0, 32, 0.5)");
+  const [fillColor, setFillColor] = useState(null);
+
+  useEffect(() => {
+    setFillColor("rgba(128, 0, 32, 0.5)");
+  });
```

Comment:
Change the useEffect() to this so that it only runs once
`useEffect(() => {
    setFillColor("rgba(128, 0, 32, 0.5)");
  },[]);`

Reply:
Sure thing! Wasn't aware of that difference.